raspios/overlay-rootfs/etc/systemd/system/dashboard.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=COGIP Dashboard
+
+Wants=network.target
+After=server.service
+
+[Service]
+Type=simple
+Restart=on-failure
+User=root
```

Comment:
We do not really care about this in our context, but just for my curiosity, does the dashboard could be run rootless ?

Reply:
Yes.